src/Models/Servers/Server.php

```diff
@@ -103,12 +103,12 @@ class Server extends Model implements Resource
     public $includedTraffic;
 
     /**
-     * @var array|\LKDev\HetznerCloud\Models\Protection
+     * @var stdClass|\LKDev\HetznerCloud\Models\Protection
```

Comment:
This looks like a bug, because it should always be an object. I will look into it.